tutorials/get-started-containers-docker/index.mdx

```diff
@@ -0,0 +1,303 @@
+
+    - `WORKDIR` to define the working directory of the Docker container
+    - `RUN` to specify a command to be executed inside the Docker container
+    - `ENV` to set an environment variable
+    - `EXPOSE` to tell the container to listen on a specified network port during its runtime
+    - `COPY`, which is similar to `ADD` and copies files to the Docker image.
+
+  Return to the terminal of your local machine for the next steps.
+
+2. Clone the **whoami** project to your local machine:
+
+  ```bash
+  git clone https://github.com/jwilder/whoami.git
+  ```
+
+3. Navigate into the `whoami` directory that has been created:
+
+  ```bash
+  cd whoami
+  ```
+
+4. Build the Docker image for the application:
+
+  ```bash
+  sudo docker build -t testwhoami .
+  ```
+
+5. Run the container image with the following command. Note that the `-d` argument tells Docker to run the contianer in the background (**d**etached mode), and the `-p` argument exposes the necessary ports for the application:
+
+  ```bash
+  sudo docker run -d -p 8000:8000 -t testwhoami
+  ```
+
+6. Carry out the following command to show all the Docker processes currently running:
+
+  ```bash
+  sudo docker ps 
+  ```
+
+  You should see the `testwhoami` image in the output, showing that the container is running.
+
+7. Open up a browser and go to the application's endpoint, which is the IP address of your local machine with the port 8000:
+
+  ```
+  http://<your-ip-address>:8000
+  ```
+
+  <Message type="tip">
+
+    If you don't know your local IP address, use the following command to find out:
+
+    ```
+    hostname --all-ip-addresses | awk '{print $1}'
+    ```
+
+  </Message>
+
+  You should see that the container image is printing out its container ID at the endpoint, with an output similar to the following:
+
+  ```
+  I'm 41ed60d6177a
+  ```
+
+  The container ID should match that which you saw with the `docker ps` command.  
+
+8. Run the following command in your terminal to shut down the container. Replace `<container-ID>` with the ID for your container.
+
+  ```
+  sudo docker kill <container-ID>
+  ```
+
+9. Run `sudo docker ps` and/or refresh the endpoint in your browser, to show that the container is no longer running.
+
+## Pushing to the Scaleway Container Registry
+
+To finish, we push the Docker images we have created to a container registry. Container registries are designed to store container images, and make them accessible to those who need them. There are many different container registry platforms, but here we use the [Scaleway Container Registry](https://www.scaleway.com/en/container-registry/).
+
+1. Open a browser and go to the [Scaleway console's Container Registry page](https://console.scaleway.com/registry/namespaces).
```

Comment:
```suggestion
1. Open a browser and go to the [Scaleway Console's Container Registry page](https://console.scaleway.com/registry/namespaces).
```

Reply:
can't remember if we already discussed this in Slack, but handbook = Scaleway console